scripts/build/builders/efr32.py

```diff
@@ -137,6 +152,52 @@ def __init__(self,
             self.extra_gn_options.append(
                 'chip_build_libshell=false enable_openthread_cli=false show_qr_code=false disable_lcd=true')
 
+        if chip_build_libshell:
+            self.extra_gn_options.append('chip_build_libshell=true')
+
+        if chip_logging == False:
+            self.extra_gn_options.append('chip_logging=false')
+
+        if chip_openthread_ftd == False:
+            self.extra_gn_options.append('chip_openthread_ftd=false')
+
+        if enable_heap_monitoring:
+            self.extra_gn_options.append('enable_heap_monitoring=true')
+
+        if enable_openthread_cli == False:
+            self.extra_gn_options.append('enable_openthread_cli=false')
+
+        if show_qr_code:
+            self.extra_gn_options.append('show_qr_code=true')
+
+        if enable_wifi:
+            self.dotfile += self.root + '/build_for_wifi_gnfile.gn'
+            if board == Efr32Board.BRD4161A:
+                self.extra_gn_options.append('is_debug=false chip_logging=false')
+            else:
+                self.extra_gn_options.append('disable_lcd=true use_external_flash=false')
```

Comment:
i m not a fan of the disable_lcd , feel backward with all the rest of the enable_something convention

Reply:
lcd is enabled by default in our configuration, only specific board require it disabled, which is why we chose that logic